cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/mapping/beans/ServiceBean.java

```diff
@@ -47,12 +50,21 @@ public void setTags(List<String> tags) {
         this.tags = tags;
     }
 
+    public Map<String, Object> getParams() {
+        return params;
+    }
+
+    public void setParams(Map<String, Object> params) {
```

Comment:
This code is not used in your program. 
Also, if I remove this method, I can still run all the tests. 

Please clarify if this is really needed.